heartbeat/Makefile

```diff
@@ -13,8 +13,7 @@ collect: imports kibana
 # Generate imports for all monitors
 .PHONY: imports
 imports:
```

Comment:
I think it's only used by heartbeat's own `collect` target so it can be removed if you want.

Reply:
Good catch, removed